network/network.go

```diff
@@ -154,6 +140,13 @@ func NewNetworkInstance(
 		didDocumentFinder:   didDocumentFinder,
 		eventPublisher:      eventPublisher,
 		storeProvider:       storeProvider,
+		selfTestDialer: tls.Dialer{
+			NetDialer: &net.Dialer{
+				Timeout:   time.Second, // TODO: what is a reasonable timeout for this? It runs every couple of seconds when the /health endpoint is called
+				LocalAddr: nil,         // TODO: should this be filled for logging purposes or something?
+			},
+			Config: &tls.Config{InsecureSkipVerify: true}, // set during Configure
```

Comment:
Why set this if (especially with an insecure value) always overridden? Then I'd prefer to have it unset.

Reply:
This is a self-check that only verifies if a connection can be made without any data exchange, so probably not that insecure? My question is _if_ we want to verify certificates in this self-check or only that the endpoint can be reached?

Reply:
NutsComm is already verified agains the certificate in the node. 
If we verify the certificate, the dialed address is matched against the certificate.